specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -336,6 +336,14 @@ _jakarta.xml.rpc.handler.MessageContext_ type.]
 
 If a container supports Java SE 11 or a newer version of the Java SE platform, than all classes and interfaces provided by the container to satisfy the platform requirements listed above, must be compiled with the Java SE 11 source and class level.
 
+[[a2841]]
+==== Platform Prospective Specifications
+
+During the development cycle for the current version of the Jakarta EE specification, the platform project considered several component specifications for inclusion in the platform. A consensus could not be reached on including these specifications in the platform. These specifications are considerd as prospects for inclusion in a future version of the Platform specification.
+
```

Comment:
I think this should have some links, e.g. to https://projects.eclipse.org/projects/ee4j.nosql and https://projects.eclipse.org/projects/ee4j.mvc .